client/src/components/Panels/Buttons/PanelViewButton.vue

```diff
@@ -2,12 +2,15 @@
     <b-dropdown
         v-b-tooltip.hover
         right
-        role="button"
         title="Show panel options"
         variant="link"
+        role="menu"
         aria-label="View all tool panel configurations"
         class="tool-panel-dropdown"
         size="sm">
+        <template v-slot:button-content>
```

Comment:
Nice, I didn't get how that was a problem with the checker website, that should fix it.